src/DependencyInjection/Core/Community.VisualStudio.Toolkit.DependencyInjection.Core.targets

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's unclear to me whether we want/need this for these new projects. @madskristensen?